android/app/src/main/java/com/recraftreactnativedatingapp/MainActivity.java

```diff
@@ -10,6 +10,6 @@
    */
   @Override
   protected String getMainComponentName() {
-    return "recraftReactNativeDatingApp";
+    return "recraftReactNativeBaseApp";
```

Comment:
Why we are changing the name again to baseApp?

Reply:
actually yesterday we have changed the names for android but the main project is cloned so when we try to change the app name in app.json it stops the whole project and give the app not register error so I have only change the name for android only for now

Reply:
because if we don't change name for android we can't upload the release apk file to play store